pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -4,111 +4,33 @@ self:
 let
   inherit (self) callPackage;
 in
-{
+lib.packagesFromDirectoryRecursive {
```

Comment:
TIL

Reply:
Yeah, we should also use this for python